docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -721,6 +761,89 @@ def uses_db_connection():
 
 If a resource you want to initialize has dependencies on other resources, those can be included in the dictionary passed to <PyObject object="build_resources"/>. For more in-depth usage, check out the [Initializing Resources Outside of Execution](/concepts/resources#initializing-resources-outside-of-execution) section.
 
+### Monitoring asset partitions with sensors
+
```

Comment:
Could you add a list of links to the subsections here? It makes it easy to see what's in this bit and jump around. Ex:

```
- [Updating a weekly asset partition when upstream daily partitions are materialized]()
- [Materializing a daily asset partition if both upstream daily partitions are materialized]()
```